WinUIGallery/ControlPages/XamlFundamentals/XamlStylesPage.xaml.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Data;
+using Microsoft.UI.Xaml.Input;
+using Microsoft.UI.Xaml.Media;
+using Microsoft.UI.Xaml.Navigation;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+
+namespace WinUIGallery.ControlPages
+{
+    public sealed partial class XamlStylesPage : Page
+    {
+        public XamlStylesPage()
+        {
+            this.InitializeComponent();
+
+            ApplyButtonStyle(MyButton);
+        }
+
+        private static void ApplyButtonStyle(Button myButton)
+        {
+            var buttonStyle = new Style(typeof(Button));
```

Comment:
I'd maybe focus here on getting a XAML defined style in code-behind as well? That feels like a use-case that is more often used?